meinberlin/apps/bplan/views.py

```diff
@@ -64,9 +66,17 @@ class BplanProjectListView(DashboardBaseMixin,
     menu_item = 'project'
 
     def get_queryset(self):
-        return super().get_queryset().filter(
+        qs = super().get_queryset().filter(
             organisation=self.organisation
         )
+        if (is_initiator(self.request.user, self.organisation)
+                or is_superuser(self.request.user)):
+            return qs
+        else:
+            return qs.filter(
+                group__id__in=[self.request.user.groups.values_list(
+                    'id', flat=True)]
+            )
```

Comment:
Im a bit unhappy with repeating this everywhere.. maybe we could have a method (sth like get_projects_for_user) and put it into a mixin?

Reply:
Hmmmmm, but as a mixin defining get_queryset, I could only use it for two of the four cases, because the others have other extra-stuff in it. Ah, right, or I could then add/filter for the other things afterwards.